scripts/profiler/stream.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+import socket
+import errno
+
+RECV_BUF_SIZE = 2**13
+
+class Stream():
+    def __init__(self, own_socket_dict, timeouts, remote_socket_dict=None):
+        self.own_sock_desc = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+        self.own_sock_desc.bind(own_socket_dict['descriptions'])
+        self.own_sock_desc.settimeout(timeouts['descriptions'])
+        self.own_sock_ev = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+        self.own_sock_ev.bind(own_socket_dict['events'])
+        self.own_sock_ev.settimeout(timeouts['events'])
```

Comment:
Could be wrapped into a function

Reply:
I don't see it. This function would have three lines, four arguments and would be executed only twice so I think there is no point in doing that.